public/extensions/door.js

```diff
@@ -0,0 +1,119 @@
+import daily from "./core.js";
+import Socket from "./socketiostate.js";
+
+let state, call;
+let socket;
+let pounced = false;
+let openCallback, closeCallback, onceOpenCallback;
+let openCallbacks = [];
+let closeCallbacks = [];
+let props = { room: "", domain: "" };
+let key = "door";
+
+let buttons = {
```

Comment:
Should we move these into the repo?

Reply:
Chad's answer while on the call: We need to pass in a publicly-available image somewhere; not wanting to couple the examples repo with the same place we are hosting it; helps with deploying this wherever and have it work out-of-the-box